src/libraries/Microsoft.PowerFx.Core/Public/Types/Serialization/FormulaTypeSchema.cs

```diff
@@ -28,6 +28,8 @@ public enum ParamType
             UntypedObject,
             EntityRecord,
             EntityTable,
+            Unknown,
+            Error
```

Comment:
To check - should we just fail serialization out-right if there are errors here?